### DISCUSSION TITLE
How to use different port rather than default 80

### DISCUSSION THREAD

**Discussion by victor10292** [firefly-iii/firefly-iii#9539] [General]
I found Firefly-iii can't use other port in Nginx. Even I configured port directive in Nginx, it doesn't work. Is there any way to change it?

**Comment by JC5** [comment]
Hi, thanks for opening a discussion.

Firefly III can use any port. There's literally no limit at all. 

Could you tell me more about your setup?

**Comment by victor10292** [comment]
I am using Nginx as web server. Here is the content of the configuration file for firefly application. You can see the listening port is set 80. It works fine. If I change it to any other port, saying 8066, the page will be forwarded to localhost:80 on which there is no application is deployed.

Alternatively, I can start a standalone server by executing `php artisan serve --port=8066`. It works perfect except using more memory. I still want to make full use of nginx and php-fpm which have been launched for other applications.

So did I make any mistake in the configuration?

```shell
server {
    listen       80;
    server_name  localhost;

    root /opt/firefly-iii/public;
    access_log  /opt/firefly-iii/var/logs/access.log;
    error_log  /opt/firefly-iii/var/logs/error.log;
    

    location / {
	try_files $uri /index.php$is_args$args;
	autoindex on;
	sendfile off;
    }

    location ~ \.php(/|$) {
	fastcgi_pass unix:/run/php/php8.4-fpm.sock;
	fastcgi_index index.php;
        fastcgi_split_path_info ^(.+\.php)(/.*)$;	
	include fastcgi_params;
	fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
    }
}
```

**Reply by JC5** [reply]
Sorry, it looks correct to me. Did you set any extra environment variables?
